proxy.go

```diff
@@ -518,12 +518,30 @@ func (rp *reverseProxy) getScope(req *http.Request) (*scope, int, error) {
 		c = rp.clusters[u.toCluster]
 		cu = c.users[u.toUser]
 	}
+
+	isTransparent := false
+	if u == nil {
+		u = rp.users[config.TransparentUser]
+		if u != nil {
+			c = rp.clusters[u.toCluster]
+			if c != nil {
+				transparent_cu := *c.users[u.toUser]
```

Comment:
let's stick to the same naming convention everywhere
```suggestion
				transparentCu := *c.users[u.toUser]
```

Reply:
Fixed